docker-compose.yml

```diff
@@ -112,6 +114,64 @@ services:
             configdb:
                 condition: service_healthy
 
+    dramatiq_task_scheduler:
+        image: observatorycontrolsystem/observation-portal:3.1.13
+        environment:
+            - DB_HOST=db
+            - DB_NAME=observationportal
+            - DB_USER=postgres
+            - DB_PASSWORD=postgrespass
+            - DEBUG=true
+            - SECRET_KEY=ocs_example_obs_portal_secret_key
+            - CONFIGDB_URL=http://configdb:7000
+            - DOWNTIMEDB_URL=http://downtime:7500
+            - ELASTICSEARCH_URL=
+            - CORS_ALLOW_CREDENTIALS=true
+            - CORS_ORIGIN_WHITELIST=http://localhost:8080,http://127.0.0.1:8080
+            - CSRF_TRUSTED_ORIGINS=localhost:8080,127.0.0.1:8080
+            - DRAMATIQ_BROKER_HOST=redis
+            - DRAMATIQ_BROKER_PORT=6379
+        mem_limit: "512m"
+        restart: always
+        command: >
+            sh -c "sleep 60 && python manage.py runscript observation_portal.task_scheduler"
```

Comment:
What's the `sleep 60` in here for?

Reply:
Oh, that's a remnant from some debugging when I was trying to rule out startup order as the cause of an issue. I've removed the sleep statements.